src/main.ts

```diff
@@ -0,0 +1,66 @@
+import { TechKidsBotGAS } from "tech-kids-bot-gas";
+import { GoogleChat } from "tech-kids-bot-gas/dist/main";
+
+
+
+// ****************************************************
+// * リマインドするメンターのメールアドレスの配列を取得
+// * A2のセルを取得し、下記のフォーマットの際で正常に機能します
+// * 
+// * 注意　　　複数のユーザーに対して送信する場合は、","で区切ってください
+// *          ","の間にスペースは入れないでください
+// * "xxxx_xxxx@ca-techkids.com,xxxx_xxxx@ca-techkids.com,xxxx_xxxx@ca-techkids.com"
+// ****************************************************
```

Comment:
JSやTSでのドキュメンテーション文字列は以下のような形式に定められています。統一してください。
(こうすることで、VSCodeなどで関数名などをマウスオーバーしたとき、そのドキュメントも表示されます)

```suggestion
/**
 * リマインドするメンターのメールアドレスの配列を取得
 * A2のセルを取得し、下記のフォーマットの際で正常に機能します
 * 
 * 注意　　　複数のユーザーに対して送信する場合は、","で区切ってください
 *          ","の間にスペースは入れないでください
 * "xxxx_xxxx@ca-techkids.com,xxxx_xxxx@ca-techkids.com,xxxx_xxxx@ca-techkids.com"
 */
```

Reply:
[320ffda](https://github.com/ca-techkids/KensyuIzumi-Remaind/pull/1/commits/320ffda0105fc5e46a5c9f0c57c86f34386100c4)
コメントの記法を修正しました